impeller/entity/contents/text_contents.cc

```diff
@@ -25,8 +25,8 @@ TextContents::TextContents() = default;
 
 TextContents::~TextContents() = default;
 
-void TextContents::SetTextFrame(const TextFrame& frame) {
-  frame_ = frame;
+void TextContents::SetTextFrame(TextFrame&& frame) {
+  frame_ = std::move(frame);
```

Comment:
Why is this better? How is the compiler not just doing this for us? haha

Reply:
If you look in the PR, there is one example where you can see SetTextFrame couldn't move the parameter since it is modified and then send to a different SetTextFrame.  So by making it an rvalue the compiler knows its safe to just move it.